lua/catppuccin/init.lua

```diff
@@ -80,20 +80,46 @@ function M.compile()
 	M.flavour = _flavour -- Restore user flavour after compile
 end
 
+---@param default? string
+local function get_flavour(default)
+	local flavour
+	if default then
+		flavour = default
+	elseif vim.g.colors_name and vim.g.colors_name == "catppuccin" then
+		-- after first time load
+		flavour = M.options.background[is_vim and vim.eval "&background" or vim.o.background]
+	else
+		-- first time load
+		flavour = M.flavour
+	end
+
+	if flavour and not M.flavours[flavour] then
+		vim.notify(
+			string.format(
+				"Catppuccin (error): Invalid flavour '%s', flavour must be 'latte', 'frappe', 'macchiato' or 'mocha'",
+				flavour
+			),
+			vim.log.levels.ERROR
+		)
+		flavour = nil
```

Comment:
```lua
flavour = "mocha"
```

I think assign mocha right here make more sense

Reply:
`M.flavour` might be nil on the initial setup, so the following line
```
return flavour or "mocha"
```
is necessary.
Of course, I can change this line to `flavour = "mocha"`, but I would like to prevent the duplication of writing "mocha" twice:)
What do you think?

Reply:
Ah sure 😅